webapp/channels/src/components/channel_members_rhs/channel_members_rhs.tsx

```diff
@@ -150,7 +150,9 @@ export default function ChannelMembersRHS({
 
             listcp.push({type: ListItemType.Member, data: member});
         }
-        setList(listcp);
+        if (JSON.stringify(list) !== JSON.stringify(listcp)) {
+            setList(listcp);
+        }
```

Comment:
if we add `list` as dependency do we still need to do this?

Reply:
Unfortunaly not working. adding list as dependency breaks the page when members button is clicked in RHS.

Reply:
I see thank you for trying it out.